### PR TITLE
Timeout based on time rather than number of retries for ASP.NET Core

### PR DESCRIPTION
**Pull request by issacnitin**

### CODE REVIEW

File: dotnet/aspnetcore/containerWithTests/Application/aspnet-core-dotnet-core.FunctionalTests/SampleFunctionalTests.cs [hunk 1]
```diff
@@ -36,8 +36,9 @@ public void TestClean()
         public void SampleFunctionalTest1()
```
Comment (sachinma) [inline dotnet/aspnetcore/containerWithTests/Application/aspnet-core-dotnet-core.FunctionalTests/SampleFunctionalTests.cs]: we are not using this sample anymore. We are likely using aspnetcore22 now

Comment (issacnitin) [reply dotnet/aspnetcore/containerWithTests/Application/aspnet-core-dotnet-core.FunctionalTests/SampleFunctionalTests.cs]: We have this sample deployed currently, shall I close the PR, if we are using aspnetcore22 from now?

Comment (sachinma) [reply dotnet/aspnetcore/containerWithTests/Application/aspnet-core-dotnet-core.FunctionalTests/SampleFunctionalTests.cs]: let me check

Comment (sachinma) [reply dotnet/aspnetcore/containerWithTests/Application/aspnet-core-dotnet-core.FunctionalTests/SampleFunctionalTests.cs]: let me check

**Pull request closed without merging**